src/main/java/com/conveyal/gtfs/loader/EntityPopulator.java

```diff
@@ -117,6 +118,16 @@
         return fareAttribute;
     };
 
+    EntityPopulator<FareRule> FARE_RULE = (result, columnForName) -> {
+        FareRule fareRule       = new FareRule();
+        fareRule.fare_id        = getStringIfPresent(result, "fare_id", columnForName);
+        fareRule.route_id       = getStringIfPresent(result, "route_id", columnForName);
+        fareRule.origin_id      = getStringIfPresent(result, "origin_id", columnForName);
+        fareRule.destination_id = getStringIfPresent   (result, "destination_id", columnForName);
+        fareRule.contains_id    = getStringIfPresent   (result, "contains_id", columnForName);
```

Comment:
Remove unneeded spaces:

```suggestion
        fareRule.destination_id = getStringIfPresent(result, "destination_id", columnForName);
        fareRule.contains_id    = getStringIfPresent(result, "contains_id", columnForName);
```

Reply:
This entity populator is no longer required, so have removed it.